### DISCUSSION TITLE
srsRAN du_example

### DISCUSSION THREAD

**Discussion by ahmedelbushra** [srsran/srsRAN_Project#467] [General Help]
I am interested in executing the ru_example file and I did run it in the "zmq_20MHz_n7" setup, but my only issue is that it keeps printing  "waiting for data" "0 out of 35600 samples has been received". How can I transmit the data from/to the srsUE if that's possible . 
I did execute the srsue with the standard ue_zmq.config file but still no connection, and I tried to change the setup tcp// ports within both but nothing seems to work. I am pretty new to this, any help would be appreciated.

**Comment by pgawlowicz** [comment]
The srsUE zmq RX port has to match the gNB zmq TX port, and vice versa.

**Reply by ahmedelbushra** [reply]
thanks alot it works fine now
